src/app/modules/user/user.routes.ts

```diff
@@ -1,23 +1,66 @@
-import { celebrate } from 'celebrate'
+import { celebrate, Joi, Segments } from 'celebrate'
 import { Router } from 'express'
 
 import * as UserController from './user.controller'
-import * as UserRules from './user.rules'
```

Comment:
is there a new preference for inlining the rules?

Reply:
Tbh, they are just used in this place, and having to refer to yet another `*.rules` file when in `*.routes` files seems a little annoying when developing or reviewing